module/zfs/zfs_ioctl.c

```diff
@@ -6888,6 +6930,11 @@ zfs_ioctl_init(void)
 	    POOL_CHECK_SUSPENDED | POOL_CHECK_READONLY, B_TRUE, B_TRUE,
 	    zfs_keys_pool_trim, ARRAY_SIZE(zfs_keys_pool_trim));
 
+	zfs_ioctl_register("scrub", ZFS_IOC_POOL_SCRUB,
```

Comment:
Is there a reason you created a new ioctl for scrubs, rather than adding an error scrub flag to the existing scrub ioctl (`ZFS_IOC_POOL_SCAN`)?